### ISSUE TITLE
`.deps.clj/1.11.1.1429/ClojureTools/TRANSACTION_START (No such file or directory)` error on start

### ISSUE THREAD

**Issue by ai212983** [cursive-ide/cursive#2867]
Not sure if its a Cursive or deps.clj bug.
On startup there's an error notification, here's relevant stacktrace from `idea.log`:
 ```
2023-12-27 16:19:32,028 [   7482]   WARN - #cursive.build.BuildSystemUtil - .deps.clj/1.11.1.1429/ClojureTools/TRANSACTION_START (No such file or directory)
java.io.FileNotFoundException: .deps.clj/1.11.1.1429/ClojureTools/TRANSACTION_START (No such file or directory)
	at java.base/java.io.FileOutputStream.open0(Native Method)
	at java.base/java.io.FileOutputStream.open(FileOutputStream.java:293)
	at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:235)
	at clojure.java.io$fn__11522.invokeStatic(io.clj:230)
	at clojure.java.io$fn__11522.invoke(io.clj:230)
	at clojure.java.io$fn__11459$G__11428__11466.invoke(io.clj:69)
	at clojure.java.io$fn__11496.invokeStatic(io.clj:166)
	at clojure.java.io$fn__11496.invoke(io.clj:166)
	at clojure.java.io$fn__11472$G__11424__11479.invoke(io.clj:69)
	at clojure.java.io$writer.invokeStatic(io.clj:119)
	at clojure.java.io$writer.doInvoke(io.clj:104)
	at clojure.lang.RestFn.invoke(RestFn.java:410)
	at clojure.lang.AFn.applyToHelper(AFn.java:154)
	at clojure.lang.RestFn.applyTo(RestFn.java:132)
	at clojure.core$apply.invokeStatic(core.clj:669)
	at clojure.core$spit.invokeStatic(core.clj:6956)
	at clojure.core$spit.doInvoke(core.clj:6956)
	at clojure.lang.RestFn.invoke(RestFn.java:425)
	at borkdude.deps$clojure_tools_install_BANG_.invokeStatic(deps.clj:557)
	at borkdude.deps$clojure_tools_install_BANG_.invoke(deps.clj:535)
	at borkdude.deps$_main$fn__343.invoke(deps.clj:926)
	at borkdude.deps$_main.invokeStatic(deps.clj:924)
	at borkdude.deps$_main.doInvoke(deps.clj:882)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.core$apply.invokeStatic(core.clj:667)
	at clojure.core$apply.invoke(core.clj:662)
	at cursive.deps_clj.shim$invoke_deps$fn__401.invoke(Unknown Source)
	at cursive.deps_clj.shim$do_invoke.invokeStatic(Unknown Source)
	at cursive.deps_clj.shim$do_invoke.invoke(Unknown Source)
	at cursive.deps_clj.shim$invoke_deps.invokeStatic(Unknown Source)
	at cursive.deps_clj.shim$invoke_deps.invoke(Unknown Source)
	at clojure.lang.Var.invoke(Var.java:393)
	at org.projectodd.shimdandy.impl.ClojureRuntimeShimImpl.invoke(ClojureRuntimeShimImpl.java:136)
	at org.projectodd.shimdandy.impl.ClojureRuntimeShimImpl.invoke(ClojureRuntimeShimImpl.java:128)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at clojure.lang.Reflector.invokeMatchingMethod(Reflector.java:167)
	at clojure.lang.Reflector.invokeInstanceMethod(Reflector.java:102)
	at cursive.embed$invoke_in.invokeStatic(embed.clj:52)
	at cursive.embed$invoke_in.doInvoke(embed.clj:46)
	at clojure.lang.RestFn.invoke(RestFn.java:490)
	at cursive.deps.project$invoke_deps$fn__20499.invoke(project.clj:223)
	at cursive.deps.project$invoke_deps.invokeStatic(project.clj:221)
	at cursive.deps.project$invoke_deps.invoke(project.clj:215)
	at cursive.deps.project$basis_for.invokeStatic(project.clj:312)
	at cursive.deps.project$basis_for.invoke(project.clj:310)
	at cursive.deps.project$project_details.invokeStatic(project.clj:453)
	at cursive.deps.project$project_details.doInvoke(project.clj:425)
	at clojure.lang.RestFn.invoke(RestFn.java:537)
	at clojure.lang.Var.invoke(Var.java:414)
	at cursive.api.DelayedFn.invoke(DelayedFn.java:56)
	at cursive.deps.DepsProjectDetails.doGetDetails(Project.kt:130)
	at cursive.deps.DepsProjectDetails.doGetDetails(Project.kt:31)
	at cursive.build.BuildSystemProjectDetails.getProjectDetails(BuildSystem.kt:45)
	at cursive.build.BuildSystemProjectsManager.reimportAllProjects$lambda$3$lambda$2(BuildSystemManager.kt:150)
	at cursive.build.BuildSystemUtil$runTask$1$1.run(BuildSystemUtil.kt:85)
	at com.intellij.openapi.progress.impl.CoreProgressManager.startTask(CoreProgressManager.java:434)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.startTask(ProgressManagerImpl.java:132)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcessWithProgressAsynchronously$5(CoreProgressManager.java:484)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$4(ProgressRunner.java:248)
	at com.intellij.openapi.progress.ProgressManager.lambda$runProcess$0(ProgressManager.java:73)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$1(CoreProgressManager.java:192)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$12(CoreProgressManager.java:610)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:685)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:641)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:609)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:78)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:179)
	at com.intellij.openapi.progress.ProgressManager.runProcess(ProgressManager.java:73)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$5(ProgressRunner.java:248)
	at com.intellij.openapi.progress.impl.ProgressRunner$ProgressRunnable.run(ProgressRunner.java:498)
	at com.intellij.util.concurrency.ChildContext$runAsCoroutine$1.invoke(propagation.kt:81)
	at com.intellij.util.concurrency.ChildContext$runAsCoroutine$1.invoke(propagation.kt:81)
	at com.intellij.util.concurrency.ChildContext.runAsCoroutine(propagation.kt:86)
	at com.intellij.util.concurrency.ChildContext.runAsCoroutine(propagation.kt:81)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$launchTask$18(ProgressRunner.java:464)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:702)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:699)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:699)
	at java.base/java.lang.Thread.run(Thread.java:840)
```

as a result, project view is empty (nothing but `deps.edn` itself, `External Libraries` and `Scratches and Consoles`).

IntelliJ IDEA 2023.3.2 (Ultimate Edition)
Build #IU-233.13135.103, built on December 20, 2023

Runtime version: 17.0.9+7-b1087.9 aarch64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 14.2.1

com.cursiveclojure.cursive (1.13.1-2023.3)

**Comment by ck** [comment]
## Issue

I am getting the same error by following the instructions to use Polylith project in IntelliJ w/ Cursive in

https://cljdoc.org/d/polylith/clj-poly/0.2.18/doc/development?q=cursive#_cursive_setup

and 

https://github.com/cursive-ide/cursive/issues/2554


Have `Resolve over whole project` enabled for Clojure.

Steps:

1. Open `New` -> `Project from Existing Sources` in IntelliJ with Cursive.
2. Select the ./deps.edn` file
3. Select `Automatically import transitive :local/root dependencies`
4. Select all Deps projects to import

I get following error notification:


```
deps.clj/1.11.1.1435/ClojureTools/TRANSACTION_START (No such file or directory)
```

## Environment

IntelliJ IDEA 2023.3.2 (Ultimate Edition)
Build #IU-233.13135.103, built on December 20, 2023
Non-Bundled Plugins:
  com.cursiveclojure.cursive (1.13.1-2023.3)
  
Clojure 1.11.1  
poly 0.2.18 (2023-12-19)

**Comment by cursive-ide** [comment]
It seems like downgrading deps.clj to v1.11.1.1386 in _Settings | Build, Execution, Deployment | Build Tools | Clojure Deps_ will work around this.

**Comment by ShakoFarhad** [comment]
I got this issue just now after installing the new Cursive update and restarting Intellij. Downgrading deps.clj fixes the issue it seems.

**Comment by jdhollis** [comment]
I have the same problem with a rather vanilla `deps.edn` project. Downgrading to v1.11.1.1386 seems to be working (though before trying that I invalidated all the caches). Will post an update if I run into any more issues.

**Comment by cursive-ide** [comment]
Fix released in 1.13.2-eap2 on 2024-01-20.

**Comment by cursive-ide** [comment]
Fix released in 1.13.2 on 2024-04-04.
